draft-ietf-quic-transport.md

```diff
@@ -2762,6 +2719,28 @@ streams as necessary in outgoing packets without losing transmission efficiency
 to underfilled packets.
 
 
+## Frame Priority {#frame-priority}
+
+Often, there will be limits on what can be transmitted as a result of connection
+flow control or the current congestion controller state.
+
+Giving preference to the transmission of its own management functions ensures
+that a protocol functions efficiently.  That is, prioritizing frames other than
+STREAM frames ensures that loss recovery, congestion control, and flow control
+operate effectively.
+
+CRYPTO frames SHOULD be prioritized over STREAM frames prior to the completion
+of the cryptographic handshake.  This includes the retransmission of the second
```

Comment:
You already effectively said this in the last paragraph, so I think that you might want to drop this paragraph.

Reply:
I think this paragraph is good to have since it is more specific. We can drop the previous paragraph, which is simply general advice.